sky/backends/local_docker_backend.py

```diff
@@ -268,8 +268,13 @@ def _execute(self,
                  handle: LocalDockerResourceHandle,
                  task: 'task_lib.Task',
                  detach_run: bool,
-                 dryrun: bool = False) -> None:
-        """ Launches the container."""
+                 dryrun: bool = False) -> Optional[int]:
```

Comment:
It should be fine to keep the original `None`? Inheriting the base class should be fine to have a subset of the return value?

Reply:
Good point! Done. Thanks!